multibase/src/Data/ByteString/BaseN.hs

```diff
@@ -0,0 +1,932 @@
+{-# LANGUAGE ConstraintKinds            #-}
+{-# LANGUAGE DataKinds                  #-}
+{-# LANGUAGE GADTs                      #-}
+{-# LANGUAGE GeneralisedNewtypeDeriving #-}
```

Comment:
British spelling was introduced in [GHC 8.6.1](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/8.6.1-notes.html#compiler), see https://travis-ci.org/oscoin/ipfs/jobs/516248105#L351

🤣

Reply:
Lol, did I unconsciously plugged a brit-spelled lang extension into the codebase? @cloudhead 's heart attack in 3..2..1 😂